Makefile.core.mk

```diff
@@ -356,18 +356,25 @@ refresh-goldens:
 
 update-golden: refresh-goldens
 
-gen: go-gen mirror-licenses format update-crds operator-proto gen-kustomize update-golden
+gen: go-gen mirror-licenses format update-crds operator-proto copy-from-istiod gen-kustomize update-golden 
 
 check-no-modify:
 	@bin/check_no_modify.sh
 
 gen-check: check-no-modify gen check-clean-repo
 
+# Copy the injection template file from istiod chart to istiod-remote chart
+copy-from-istiod:
```

Comment:
Sorry for the drive-by review ... can we rename this target? The current name is non-descriptive. Maybe something like `istiod-remove-charts`?

Reply:
np!  this doesn't generate istiod remote chart... it is just copy 2 files over.  generate the file is part of the gen-kustomize

Reply:
`gen-sync-remote-templates` could be a better make target.

Reply:
ok 2 people have issue with this name, I've renamed :)